CONTRIBUTING.rst

```diff
@@ -39,3 +39,9 @@ Checklist
   therein.
 * Try to include tests!
 * Have fun!
+
+Translations
+++++++++++++
+
+If you're talented enough to speak another language, we'd love you to contribute a 
+translation to the editor. Learn more about how this works in the `translation release strategy <https://github.com/bbcmicrobit/PythonEditor/docs/TRANSLATIONS.MD>`_
```

Comment:
This URL gives a 404.

Reply:
the link would only work post merge, i've removed it for now